roles/pulp_common/vars/main.yml

```diff
@@ -24,4 +24,11 @@ __pulp_common_pulp_settings_defaults:
   public_key_path: "{{ pulp_certs_dir }}/token_public_key.pem"
   token_server: "https://{{ ansible_facts.fqdn }}/token"
   token_signature_algorithm: ES256
-__pulp_common_merged_pulp_settings: "{{ __pulp_common_pulp_settings_defaults|combine(pulp_settings, recursive=True) }}"
+__pulp_common_pulp_settings_fips_defaults:
+  allowed_content_checksums:
+    - sha1
+    - sha224
+    - sha256
+    - sha384
+    - sha512
+__pulp_common_merged_pulp_settings: "{{ __pulp_common_pulp_settings_defaults | combine(ansible_fips | ternary(__pulp_common_pulp_settings_fips_defaults, {}), pulp_settings, recursive=True) }}"
```

Comment:
Does this allow users to set allowed_content_checksums even if ansible_fips==False?
If ansible_fips==True and a user did not provide any allowed_content_checksums customization, does it use L#28 - L#33?
If ansible_fips==True and a user does provide customization, does the user's customization overwrite or are they merged?

Reply:
Yes!, yes!, yes? (need to check to be sure...).

Reply:
Aaaand Yes! The dicts are merged recursive, but lists are still overwritten as is and not merged.

Reply:
Great then I think this aspect is good to go. Practically speaking users can override if they don't want sha1, but we know that md5 has to be removed so we should take that helpful step and carry that default for ansible_fips==True environments. Thank you for testing!